### PR TITLE
Feature/log route requests

### PR DESCRIPTION
**Pull request by andybbruno**
logics for the handling of the `route` attribute

### CODE REVIEW

File: nephthys/clients/requests.py [hunk 1]
```diff
@@ -143,9 +149,22 @@ def send(self, request, **kwargs):
 
         return response
 
+    def request(self, method, url, **kwargs):
+
+        if "route" in kwargs:
```
Comment (NMassa) [inline nephthys/clients/requests.py:156]: you can use kwargs.pop("route", None)

File: setup.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def make_long_description():
     description="Advanced Python Logger",
     long_description=make_long_description(),
     long_description_content_type="text/markdown",
-    version="1.0.1",
+    version="1.0.2",
```
Comment (NMassa) [inline setup.py:17]: please don't change the version here, the repo owner should do this once the PR is merged with a dedicated commit and tag

**Review by NMassa — CHANGES_REQUESTED** [review]

File: nephthys/clients/requests.py (hunk 1 above)
Comment (andybbruno) [reply nephthys/clients/requests.py]: corrected!

File: setup.py (hunk 2 above)
Comment (andybbruno) [reply setup.py]: fixed this as well

**Pull request merged**